venue/components/InviteForm.vue

```diff
@@ -59,9 +59,16 @@ export default {
         referralCode: this.$store.state.user.referral_code
       });
     }
-    data.referralLink += data.referralCode;
     return data;
   },
+  mounted() {
+    this.referralLink =
```

Comment:
Constructing a URL like this seems error prone. For example, does `location.protocol` really already contain the `:`? Also does this work if there is a port specified?

Reply:
`location.protocol` already contains the `:`
`location.host` that I am using contains the hostname, `:` and the port of URL

Reply:
Cool, ok!